### PR TITLE
Add L0 test to protect end-to-end ensemble model

### PR DESCRIPTION
**Pull request by pskiran1**

### CODE REVIEW

File: qa/L0_ensemble_model/check_results.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        Check that each model was profiled the number of times
+        corresponding with batch size and concurrency combinations
+
+        (No model config parameter combos expected here!)
+        """
+
+        with open(self._analyzer_log, "r") as f:
+            log_contents = f.read()
+
+        expected_min_num_measurements = 20
```
Comment (nv-braf) [inline qa/L0_ensemble_model/check_results.py:52]: When you ran this on average how many measurements are taken?

File: qa/common/util.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ ANALYZER_LOG=${ANALYZER_LOG:="$LOGS_DIR/test.log"}
 
 mkdir -p $LOGS_DIR
 
-create_logs_dir() {
```
Comment (nv-braf) [inline qa/common/util.sh:25]: Why was this change made?

Comment (pskiran1) [reply qa/common/util.sh:25]: It seems that I forgot to include the "function" keyword when I implemented the function. I have now added it.

File: qa/L0_ensemble_model/check_results.py (hunk 1 above)
Comment (pskiran1) [reply qa/L0_ensemble_model/check_results.py:52]: I observed measurements ranging from 30 to 45 during the test runs. I considered the minimum and maximum counts mentioned above from the [L0_quick_search](https://github.com/triton-inference-server/model_analyzer/tree/main/qa/L0_quick_search) test.

**Review by nv-braf — APPROVED** [review]

**Pull request merged**